3회차/문재윤/3_Flatten.py

```diff
@@ -1,3 +1,32 @@
 import sys
 
 sys.stdin = open("_Flatten.txt")
+
+
+um = 10 
+    
+for num in range(1,um+1):
+    dump = int(input())
+
+    li = input().split()
+    li = list(map(int,li))
+
+    for i in range(dump):
+        li.sort()
+        a = li.pop()
+        li.sort(reverse =True)
+        b = li.pop()
```

Comment:
`sort`를 해주면 제일 앞은 최솟값을 가지므로, 다시 `sort`를 하지 않고 해결하는 방법이 있어요!
```suggestion
        li.sort()
        a = li.pop()
        b = li.pop(0)
```